cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,11 +19,11 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
-  typeEmail(email) {
+  typeEmail() {
     this.emailField.type(email, { force: true });
   }
```

Comment:
The `typeEmail` method is missing the `email` parameter. You should add `email` as a parameter to this method to allow it to type the email into the email field.